Source/CBL_Replicator.m

```diff
@@ -341,13 +341,6 @@ - (void) start {
                  }
                 ];
 
-    // If client didn't set an authorizer, use basic auth if credential is available:
-    if (!_authorizer) {
-        _authorizer = [[CBLBasicAuthorizer alloc] initWithURL: _remote];
```

Comment:
One thing that I should ask. Do we intend to use CBLBasicAuthorizer for digest as well? If the answer is Yes, then my comment above doesn't matter because we can't know in advance if it's basic or digest from CBLBasicAuthorizer as of now.